server/container/guice/cassandra-rabbitmq-guice/pom.xml

```diff
@@ -168,12 +197,48 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-guice-webadmin</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-webadmin-data</artifactId>
```

Comment:
james-server-guice-webmin-common ? I'm not saying to change it for this PR but I think it would be interesting.

Reply:
Again one might compose only some subparts of the webAdmin. An example of this is jpa-smtp that drops all routes related to the mailbox, to JMAP, etc...